### PR TITLE
Describe how to run gerrit pipeline on github PR.

### PR DESCRIPTION
**Pull request by m-kostrzewa**
Introduced alongside: https://github.com/Juniper/contrail-windows-ci/pull/138

### CODE REVIEW

File: doc/continuous_deployment/Run_gerrit_pipeline_on_github_PR.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+1. Clone Jenkins job `winci-server2016-devel` to some temporary `tmp-devel-gerrit-check`
+2. Configure `tmp-devel-gerrit-check`:
+* Properties content: BRANCH_NAME={branch to test}
+* Branches to build: {branch to test}
+3. Schedule `tmp-devel-gerrit-check` job manually with parameters:
+* `ZUUL_PROJECT=Juniper/contrail-controller`
+* `ZUUL_BRANCH=master`
+* `ZUUL_REF=None`
+* `ZUUL_URL=http://10.84.12.75/merge-warrior`
+* `ZUUL_UUID=$someRandomUUID`
```
Comment (mclapinski) [inline doc/continuous_deployment/Run_gerrit_pipeline_on_github_PR.md]: And yet you used `NO_ZUUL_UUID_SET` in your job. So which one is it - random uuid or NO_ZUUL_UUID_SET?

Comment (m-kostrzewa) [reply doc/continuous_deployment/Run_gerrit_pipeline_on_github_PR.md]: I don't think it matters - it's treated like normal string.

@sodar does it matter?

Comment (mclapinski) [reply doc/continuous_deployment/Run_gerrit_pipeline_on_github_PR.md]: I don't think it matters but you should decide on one - not write something in docs and use it some other way. If there's no need for developer to generate UUID then you should say that in docs.

Comment (sodar) [reply doc/continuous_deployment/Run_gerrit_pipeline_on_github_PR.md]: It does. Log path on log server is constructed from `ZUUL_UUID`.

Comment (sodar) [reply doc/continuous_deployment/Run_gerrit_pipeline_on_github_PR.md]: Here are your logs: http://logs.opencontrail.org/winci/NO_ZUUL_UUID_SET/

Comment (sodar) [reply doc/continuous_deployment/Run_gerrit_pipeline_on_github_PR.md]: Sorry for spam as it's not related to this PR, but we should change this behaviour. I mean, maybe random generated log path would be better. Then `ZUUL_UUID` would have more clearly defined purpose.

Comment (m-kostrzewa) [reply doc/continuous_deployment/Run_gerrit_pipeline_on_github_PR.md]: What should I put there for now then? @sodar

Comment (sodar) [reply doc/continuous_deployment/Run_gerrit_pipeline_on_github_PR.md]: I would go with generating random uuid.

File: doc/continuous_deployment/Run_gerrit_pipeline_on_github_PR.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+1. Clone Jenkins job `winci-server2016-devel` to some temporary `tmp-devel-gerrit-check`
+2. Configure `tmp-devel-gerrit-check`:
+* Properties content: BRANCH_NAME={branch to test}
+* Branches to build: {branch to test}
+3. Schedule `tmp-devel-gerrit-check` job manually with parameters:
+* `ZUUL_PROJECT=Juniper/contrail-controller`
+* `ZUUL_BRANCH=master`
+* `ZUUL_REF=None`
+* `ZUUL_URL=http://10.84.12.75/merge-warrior`
+* `ZUUL_UUID= (some random UUID - it will determine path on the logserver)`
```
Comment (mclapinski) [inline doc/continuous_deployment/Run_gerrit_pipeline_on_github_PR.md:19]: should be `it will be used to determine`

**Review by mclapinski — APPROVED** [review]

**Pull request merged**